src/views/ShoppingList.test.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ShoppingList from './ShoppingList';
+
+it('should render the Shopping List view', () => {
+  const { container } = <ShoppingList />;
```

Comment:
`render` needs to be imported from '@testing-library/react' and called with your component here, otherwise you will get a false passing test because the snapshot will be 'undefined'